common/ledger/util/leveldbhelper/leveldb_provider_test.go

```diff
@@ -191,6 +191,125 @@ func TestBatchedUpdates(t *testing.T) {
 	}
 }
 
+func TestDeleteAll(t *testing.T) {
+	env := newTestProviderEnv(t, testDBPath)
+	defer env.cleanup()
+	p := env.provider
+
+	db1 := p.GetDBHandle("db1")
+	db2 := p.GetDBHandle("db2")
+	db3 := p.GetDBHandle("db3")
+	db4 := p.GetDBHandle("db4")
+	for i := 0; i < 20; i++ {
+		db1.Put([]byte(createTestKey(i)), []byte(createTestValue("db1", i)), false)
+		db2.Put([]byte(createTestKey(i)), []byte(createTestValue("db2", i)), false)
+		db3.Put([]byte(createTestKey(i)), []byte(createTestValue("db3", i)), false)
+	}
+	// db4 is used to test remove when multiple batches are needed (each long key has 125 bytes)
+	for i := 0; i < 10000; i++ {
+		db4.Put([]byte(createTestLongKey(i)), []byte(createTestValue("db4", i)), false)
+	}
+
+	rangeTestCases := []struct {
+		db             *DBHandle
+		startKey       []byte
+		endKey         []byte
+		expectedKeys   []string
+		expectedValues []string
+	}{
```

Comment:
- This struct is not actually used as a test case, can you rename this as `expectedSetup`
- `startKey` and `endKey` can be removed from this struct, as they are always `nil`